packages/psss/src/containers/Panels/AlertsPanel.js

```diff
@@ -88,24 +88,37 @@ AlertsPanelProvider.propTypes = {
 
 export const AlertsPanel = React.memo(() => {
   const { data: panelData, isOpen, setIsOpen } = useContext(AlertsPanelContext);
-  const { organisationUnit: countryCode, period, syndrome, syndromeName } = panelData;
+  const { id: alertId, organisationUnit: countryCode, period, syndrome, syndromeName } = panelData;
   const alert = { period, organisationUnit: countryCode, syndrome };
-
-  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isArchiveAlertModalOpen, setIsArchiveAlertModalOpen] = useState(false);
+  const [isCreateOutbreakModalOpen, setIsCreateOutBreakModalOpen] = useState(false);
   const [activeIndex, setActiveIndex] = useState(0);
   const notesState = useFetch(getAlertsMessages);
   const activityState = useFetch(getActivityFeed);
   const countryName = useSelector(state => getCountryName(state, countryCode));
 
   const handleChange = option => {
     // TODO handle changes other than creating an outbreak
```

Comment:
Perhaps we need to update this comment to "TODO allow outbreaks to be created"... Or we can add the case for `Outbreaks` - users shouldn't be able to select this option via the UI anyway